### PR TITLE
DataGrid - Pager - Add aria- attributes (T1109686)

### PR DESCRIPTION
**Pull request by Tucchhaa**

### CODE REVIEW

**Review by pomahtri — DISMISSED** [review]
~~translation for pager's label will be added in next PR~~

File: js/renovation/ui/pager/common/light_button.tsx [hunk 1]
```diff
@@ -18,6 +18,7 @@ export const viewFunction = ({
     tabIndex={tabIndex}
     role="button"
     aria-label={label}
+    aria-current={selected ? 'page' : undefined}
```
Comment (MikeVitik) [inline js/renovation/ui/pager/common/light_button.tsx:21]: Add label localization

File: js/renovation/ui/pager/pages/__tests__/page.test.tsx [hunk 2]
```diff
@@ -20,6 +20,24 @@ describe('Small pager pages', () => {
     });
   });
 
+  // T1109686
+  it('pageIndexes: check aria-current attribute', () => {
```
Comment (MikeVitik) [inline js/renovation/ui/pager/pages/__tests__/page.test.tsx:25]: Split this test into two parts. One should test page and another should test light button

File: js/renovation/ui/pager/__tests__/content.test.tsx [hunk 3]
```diff
@@ -345,6 +347,12 @@ describe('PagerContent', () => {
       expect(component.pageIndexSelectorVisible).toBe(true);
     });
 
+    it('aria', () => {
```
Comment (MikeVitik) [inline js/renovation/ui/pager/__tests__/content.test.tsx:352]: Add check messageLocalization.format is called

**Review by MikeVitik — CHANGES_REQUESTED** [review]

**Review by MikeVitik — DISMISSED** [review]

**Review by MikeVitik — APPROVED** [review]

**Pull request merged**